app_dart/bin/server.dart

```diff
@@ -67,17 +73,31 @@ Future<void> main() async {
         githubStatusService,
         githubChecksService,
       ),
-      '/api/push-build-status-to-github': PushBuildStatusToGithub(config, authProvider),
-      '/api/push-gold-status-to-github': PushGoldStatusToGithub(config, authProvider),
-      '/api/push-engine-build-status-to-github': PushEngineStatusToGithub(config, authProvider),
-      '/api/refresh-chromebot-status': RefreshChromebotStatus(config, authProvider),
+      '/api/push-build-status-to-github':
+          PushBuildStatusToGithub(config, authProvider),
+      '/api/push-gold-status-to-github':
+          PushGoldStatusToGithub(config, authProvider),
+      '/api/push-engine-build-status-to-github':
+          PushEngineStatusToGithub(config, authProvider),
+      '/api/refresh-chromebot-status':
+          RefreshChromebotStatus(config, authProvider),
       '/api/refresh-github-commits': RefreshGithubCommits(config, authProvider),
       '/api/refresh-cirrus-status': RefreshCirrusStatus(config, authProvider),
       '/api/reserve-task': ReserveTask(config, authProvider),
-      '/api/reset-devicelab-task': ResetDevicelabTask(config, authProvider),
+      '/api/reset-devicelab-task': ResetDevicelabTask(
+        config,
+        authProvider,
+      ),
+      '/api/reset-try-task': ResetTryTask(
```

Comment:
Could this be used to re-run prod task as well? If yes, then `reset-luci-task`?

Reply:
They actually require different code paths and different parameters.